cms/jinja2/templates/pages/topic_page.html

```diff
@@ -1,28 +1,24 @@
 {% extends "templates/base_page.html" %}
+{% from "components/hero/_macro.njk" import onsHero %}
 {% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
 {% from "components/document-list/_macro.njk" import onsDocumentList %}
 
 {% block header_area %}
-    <div class="topic-header">
-        <div class="topic-header__circles" aria-hidden="true">
-            <div class="topic-header__circle-one"></div>
-            <div class="topic-header__circle-two"></div>
-            <div class="topic-header__circle-three"></div>
-        </div>
-        <div class="ons-container">
-            {% include "templates/components/navigation/breadcrumbs.html" %}
-            <p class="topic-header__doc-type">{{ _("Topic") }}</p>
-
-            <div class="ons-grid">
-                <div class="ons-grid__col ons-col-8@m ">
-                    <h1 class="ons-u-fs-3xl common-header__heading">
-                        {{ page.title }}
-                    </h1>
-                    <div class="common-header__summary">{{ page.summary|richtext }}</div>
-                </div>
-            </div>
-        </div>
-    </div>
+    {# fmt:off #}
+        {{
+            onsHero({
+                "topic": _("Topic"),
+                "title": page.title,
+                "variants": 'pale-blue',
+                "text": page.summary|richtext(),
+                "wide": true,
+                "breadcrumbs": {
+                    "ariaLabel": _("Breadcrumbs"),
```

Comment:
the trananslated string may need to be pulled to a `{%- set breadcrumbs_label = _("Breadcrumbs") -%}` bit to enable Django's translation mechanism pick it up and extract in the translation files. cc @MaciekBaron

Reply:
Ah ok thanks for the heads up, does that same logic apply to the other translatable params e.g. topic?

Reply:
Have spoken with @MaciekBaron and have updated all the translatable strings now